xslt/Image.xsl

```diff
@@ -8,17 +8,25 @@
     <xsl:import href="ObjectiveSettings.xsl"/>
     <xsl:import href="Pixels.xsl"/>
 
-    <!-- /Metadata/Information/Image/AcquisitionDataAndTime => /OME/Image/@AcquisitionDate   -->
-    <xsl:template match="AcquisitionDateAndTime">
-        <xsl:element name="ome:AcquisitionDate">
-            <xsl:value-of select="."/>
-        </xsl:element>
-    </xsl:template>
-
     <xsl:variable name="img" select="/ImageDocument/Metadata/Information/Image"/>
 
     <xsl:template name="common_image_contents">
-        <xsl:apply-templates select="$img/AcquisitionDateAndTime"/>
+        <xsl:param name="idx"/>
+        <xsl:choose>
+            <!-- /Subblocks/Subblock/METADATA/Tags/AcquisitionTime => /OME/Image/@AcquisitionDate   -->
+            <xsl:when test="$subblocks/Subblock[@S=$idx]/METADATA/Tags/AcquisitionTime">
+                <xsl:element name="ome:AcquisitionDate">
+                    <xsl:value-of select="$subblocks/Subblock[@S=$idx]/METADATA/Tags/AcquisitionTime"/>
+                </xsl:element>
+            </xsl:when>
+            <!-- /Metadata/Information/Image/AcquisitionDataAndTime => /OME/Image/@AcquisitionDate   -->
+            <xsl:when test="$img/AcquisitionDateAndTime">
```

Comment:
Is this effectively a fall-back in case the image doesn't have any Subblocks?

Reply:
It is exactly that!